docs/observability-prometheus-metrics-in-jetstream-server.md

```diff
@@ -48,4 +48,38 @@ jetstream_prefill_backlog_size{id="SOME-HOSTNAME-HERE>"} 0.0
 # HELP jetstream_slots_used_percentage The percentage of decode slots currently being used
 # TYPE jetstream_slots_used_percentage gauge
 jetstream_slots_used_percentage{id="<SOME-HOSTNAME-HERE>",idx="0"} 0.04166666666666663
-```
\ No newline at end of file
+```
+
+## Observe metrics on GKE clusters
+
+The following applies only for Jetstream deployed on a GKE cluster. Currently [Google Cloud Managed Service for Prometheus](https://cloud.google.com/stackdriver/docs/managed-prometheus) is enabled by default on all GKE clusters, it determines scrape targets via the [PodMonitoring](https://github.com/GoogleCloudPlatform/prometheus-engine/blob/v0.10.0/doc/api.md#podmonitoring) custom resource, consequently one must be applied to your cluster, the following .json manifest can be used as a template:
```

Comment:
How does users/customers use this .json manifest? Is this a required step to access metrics on GCP console?

Reply:
The podmonitoring is essentially directions to GCP managed prometheus on where to scrape metrics from so its neccesary, more about it [here](https://cloud.google.com/stackdriver/docs/managed-prometheus/setup-managed#gmp-pod-monitoring)

Reply:
Could you add this to the README?